files/image_config/midplane-network/dummy-midplane.network

```diff
@@ -0,0 +1,8 @@
+# The systemd-networkd requires that the bridge interface(bridge-midplane) has at least one member interface, otherwise the IP address will not be configured.
```

Comment:
@Pterosaur is this a known issue? Can you quote reference here?

Reply:
Add a reference. 
Actually, I didn't find an official doc about this issue. But I got this issue in my testbed and found many people were discussing this. Meanwhile, I didn't see any side-effect with an extra dummy interface and I found some popular projects,like QUEM, did add a dummy interface into their pre-defined bridge.